doc/encoding.md

```diff
@@ -117,7 +117,7 @@ Fields are positioned within the struct according to an algorithm with the follo
 
 Field offsets are computed by the Cap'n Proto compiler.  The precise algorithm is too complicated
 to describe here, but you need not implement it yourself, as the compiler can produce a compiled
-schema format which includes offset information.
+schema format which includes offset information, e.g. by running `capnp compile -ocapnp myschema.capnp`.
```

Comment:
Hmm, this is somewhat awkward in that the paragraph is really talking about programmatic access to the offsets via the compiled schemas / CodeGeneratorRequest, but `-ocapnp` produces human-readable debug output. I'm not sure this is the right place to mention `-ocapnp`, but if we do want to mention it, we should also describe in more detail how to access the compiled schemas programmatically.

Reply:
Ah, right.  I wasn't sure whether the "compiled schemas" referred to the "-ocapnp" output or something more general.  I updated the text.